src/localplayer.cpp

```diff
@@ -95,62 +91,21 @@ LocalPlayer::~LocalPlayer()
 static aabb3f getTopBoundingBox(const std::vector<aabb3f> &nodeboxes)
 {
 	aabb3f b_max;
-	b_max.reset(-BS, -BS, -BS);
-	for (std::vector<aabb3f>::const_iterator it = nodeboxes.begin();
-			it != nodeboxes.end(); ++it) {
-		aabb3f box = *it;
-		if (box.MaxEdge.Y > b_max.MaxEdge.Y)
-			b_max = box;
-		else if (box.MaxEdge.Y == b_max.MaxEdge.Y)
-			b_max.addInternalBox(box);
-	}
-	return aabb3f(v3f(b_max.MinEdge.X, b_max.MaxEdge.Y, b_max.MinEdge.Z), b_max.MaxEdge);
-}
 
-#define GETNODE(map, p3, v2, y, valid) \
-	(map)->getNodeNoEx((p3) + v3s16((v2).X, y, (v2).Y), valid)
+	if (nodeboxes.size() == 0) {
```

Comment:
Using the old code it's no longer possible to sneak on the most outer side of stairs or similar formed collision boxes that aren't flat since the movement is limited to the top box even if there are more collision boxes below.
However, will rename this function.